### PR TITLE
Add request id in get plans API context

### PR DESCRIPTION
**Pull request by weihesdlegend**
## Description
Clearly describe changes in your PR, new feature or bug fixes

## Root Cause
Briefly describe the root cause and analysis of the problem

## Solution
Describe your code changes in detail for reviewers. Explain the solution and how it fixes the issue.
For new features, describe the high-level ideas.

## Testing
Did you add any new test for this change, performed manual integration tests, or both?

## Final Checks
- [ ] Have you removed commented code ?
- [ ] Have you used gofmt to format your code ?
- [ ] You have added unit tests

### CODE REVIEW

File: config/config.yml [hunk 1]
```diff
@@ -6,4 +6,3 @@ server:
       - formatted_address
       - adr_address
       - url
-      - user_ratings_total
```
Comment (weihesdlegend) [inline config/config.yml:9]: To save cost for getting atmosphere data.

File: planner/planner_API.go [hunk 2]
```diff
@@ -323,20 +325,21 @@ func (planner *MyPlanner) getPlanningApi(context *gin.Context) {
 		planningReq.SlotRequests[slotReqIdx].Location = cityCountry // set to the same location from URL
 	}
 
-	planningResp := planner.Planning(context, &planningReq, username)
+	c := context.WithValue(ctx, "request_id", requestId)
```
[the remark this answers is not in the document]
Comment (weihesdlegend) [reply planner/planner_API.go:328]: No, the ctx is of gin.Context type, not the same as context.Context which we need here.

**Review by xtpitt — APPROVED** [review]
Nice

**Review by xtpitt — APPROVED** [review]
NICE

**Pull request merged**